### ISSUE TITLE
Increased inference time after pruning channels

### ISSUE THREAD

**Issue by StefanL19** [BenWhetton/keras-surgeon#10]
I am trying to prune channels of a model which contains Convolutional layers (it contains other layers as well such as Batch Normalization, Dropout). I am pruning the channels of convolutional layers only, but the more I increase the pruning rate, the more increase in the inference time I get. This happens although the pruned model has less weights (even up to 90% less) than the original one. Are familiar with such a problem?

**Comment by pribadihcr** [comment]
@StefanL19,
Why did you close this issue. Is it any reasonable answer?. I am courius too.

**Comment by StefanL19** [comment]
It turned out to be a bug in the code I have written on top of ResNet50 (since I am using it as a backbone for another model). The pruning mechanisms gives expected results on the ResNet50 model.
